src/components/Feed/Feed.tsx

```diff
@@ -4,30 +4,41 @@ import styled from 'styled-components';
 
 import breakpoints from '~/components/core/breakpoints';
 import { VStack } from '~/components/core/Spacer/Stack';
+import { FEED_MODE_KEY } from '~/constants/storageKeys';
 import { FeedViewerFragment$key } from '~/generated/FeedViewerFragment.graphql';
+import usePersistedState from '~/hooks/usePersistedState';
 
+import FeedModeSelector from './FeedModeSelector';
 import GlobalFeed from './GlobalFeed';
 import ViewerFeed from './ViewerFeed';
 
 export type FeedMode = 'FOLLOWING' | 'WORLDWIDE' | 'USER';
 
 type Props = {
   queryRef: FeedViewerFragment$key;
-  setFeedMode: (mode: FeedMode) => void;
-  feedMode: FeedMode;
 };
 
-export default function Feed({ queryRef, feedMode, setFeedMode }: Props) {
+export default function Feed({ queryRef }: Props) {
```

Comment:
moved the logic that maintains the Feed Mode state from the main navbar into this file, since everything Feed related is now contained within the feed page